fbthrift-transport/README.md

```diff
@@ -0,0 +1,19 @@
+# fbthrift-transport
+
+* [Cargo package](https://crates.io/crates/fbthrift-transport)
+
+The origin copyright in https://github.com/bk-rs/fbthrift-transport
```

Comment:
If you want to use the `https://github.com/bk-rs/fbthrift-transport`, you can add it as a dependence but no directly copy the source code. follow-up maintenance is not convenient.

Reply:
We need modify it for the fbthrift and interface require.